packages/core/src/api.ts

```diff
@@ -239,9 +239,12 @@ export const addAdapter = async (
     throw new Error('No adapter available for this service')
   }
   await workspace.addService(adapterName)
-  const defaultConfig = getDefaultAdapterConfig(adapterName)
-  if (!_.isUndefined(defaultConfig)) {
-    await workspace.updateServiceConfig(adapterName, defaultConfig)
+
+  if (_.isUndefined((await workspace.servicesConfig([adapterName]))[adapterName])) {
```

Comment:
nit
```suggestion
  if (_.isEmpty(await workspace.servicesConfig([adapterName]))) {
```
if this works I think it is a bit easier to understand

Reply:
It doesn't work..